source/2_platform/rex_windows/src/entrypoint.cpp

```diff
@@ -8,10 +8,20 @@
 #include "rex_windows/log.h"
 #include "rex_windows/platform_creation_params.h"
 
+#include "rex_std/iostream.h"
+
 #define NOMINMAX
 #include <Windows.h>
 #include <processenv.h>
 
+//-------------------------------------------------------------------------
+s32 report_crash(LPEXCEPTION_POINTERS ExceptionInfo)
+{
+  rsl::cout << "We crashed!\n";
```

Comment:
If would use _set_se_translator here ( https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/set-se-translator?view=msvc-170 ) to receive a more meaningfull description.

Reply:
my implementation is based on how Unreal does it, you can get the same info using the link above as how I'm doing it, one is the C way, the other the C++ way I think.
We discussed the this implementation is fine